server/models/tags/getTag.js

```diff
@@ -0,0 +1,26 @@
+const { pool } = require('../database')
+
+/**
+ *
+ * @param {*} param0
+ * @param {Object} param0.query
+ * @return {Promise}
+ */
+
+function getTag({ query }) {
+  return new Promise((resolve, reject) => {
+    const keyword = query.keyword || ''
+    pool.query(
+      `SELECT * FROM tags WHERE name LIKE ?`,
+      [`%${keyword}%`],
```

Comment:
You do not need to escape strings this way, it is automatically done when this syntax is used.

Refer [this link](https://github.com/mysqljs/mysql#escaping-query-values)

Reply:
you mean doing this:
`SELECT * FROM tags WHERE name LIKE %${keyword}%` ?

Reply:
This is fine, if it works then no need to change